kyuubi-server/pom.xml

```diff
@@ -78,6 +78,12 @@
             <version>${project.version}</version>
         </dependency>
 
+        <dependency>
+            <groupId>org.apache.kyuubi</groupId>
+            <artifactId>kyuubi-relocated-hive-metastore-client</artifactId>
+            <version>${kyuubi-relocated.version}</version>
+        </dependency>
```

Comment:
can we add comment to say it is only used for hive delegation token ?

Reply:
see https://github.com/apache/kyuubi-shaded/pull/35

so far, it's only for token, but it will be changed over time